ansible/artifacts-download.yml

```diff
@@ -4,7 +4,7 @@
   vars_files:
     - "{{inventory_dir}}/secrets.yml"
   environment:
-    AZURE_STORAGE_ACCOUNT: "{{sunbird_artifact_storage_account_name}}"
-    AZURE_STORAGE_SAS_TOKEN: "{{sunbird_artifact_storage_account_sas}}"
+    AZURE_STORAGE_ACCOUNT: "{{ artifact_azure_account_name }}"
```

Comment:
we aren't using these variables anymore, do revert back to sunbird_artifact_storage_account_name and sunbird_artifact_storage_account_key for all the below commits

Reply:
updated to new variables now.